msicreator/createmsi.py

```diff
@@ -31,6 +31,11 @@ def __init__(self, dirs, files):
         self.dirs = dirs
         self.files = files
 
+class UIGraphics:
```

Comment:
UI graphics feature was missing in the previous createmsi.py script, so I added it back.